less/pages/thank-you.less

```diff
@@ -14,21 +14,21 @@
   display: inline;
   .header {
     text-align: center;
-    margin-bottom: 0;
+    margin-bottom: 20px;
     margin-top: 0;
-    padding-top: 22px;
+  max-width:none;
```

Comment:
Two spaces worth of indent. And put a space between max-width: and none, so like this: `max-width: none;`